onnxruntime/python/tools/symbolic_shape_infer.py

```diff
@@ -1075,6 +1076,17 @@ def _infer_Pool(self, node):
                 helper.make_tensor_value_info(o, vi.type.tensor_type.elem_type,
                                               get_shape_from_sympy_shape(sympy_shape)))
 
+    def _infer_aten_bitwise_or(self, node):
+        sympy_shape = self._get_sympy_shape(node, 0)
+        other = self._try_get_value(node, 1)
+        assert other is not None
+
+        if node.output[0]:
+            vi = self.known_vi_[node.output[0]]
+            vi.CopyFrom(
```

Comment:
can you check if this op is supposed to support broadcasting? if yes, we need to infer output shape from both of the input shapes

Reply:
> can you check if this op is supposed to support broadcasting? if yes, we need to infer output shape from both of the input shapes

Added broadcast in infer